dask_ml/wrappers.py

```diff
@@ -323,6 +323,11 @@ class Incremental(ParallelPostFit):
            a single NumPy array, which may exhaust the memory of your worker.
            You probably want to always specify `scoring`.
 
+    max_iter : int, default 1
```

Comment:
Should the default be 1? That seems low. I'd choose a higher value, at least 10.